src/FormDataFactory/DemoFormDataFactory.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Website\FormDataFactory;
+
+use Nette\Utils\FileSystem;
+use Rector\Website\Entity\RectorRun;
+use Rector\Website\ValueObject\DemoFormData;
+
+final class DemoFormDataFactory
+{
+    public function createFromRectorRun(?RectorRun $rectorRun): DemoFormData
+    {
+        if ($rectorRun) {
+            return new DemoFormData($rectorRun->getContent(), $rectorRun->getConfig());
+        }
+
+        // default values
+        $demoContent = FileSystem::read(__DIR__ . '/../../data/DemoFile.php');
```

Comment:
Ultimately these paths could be passed as parameters via constructor, so it is easier to test this class.

Reply:
Parameters suggest it's configurable, which is not.
Also `FileSystem` is 3rd party code, we don't need to test it.

Might be constants though to make it more clear :+1: Thanks for the hint

Reply:
Also, parameters or compositive paths (`$someDir . '/path.php') suck at file jumping:

![jump](https://user-images.githubusercontent.com/924196/73592351-a1c7e900-44f9-11ea-98ee-ea34337acaf4.gif)